script.js

```diff
@@ -0,0 +1,27 @@
+// one-var
+
+var a = 10;
+
+const value = 10;
+value++;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['value' is constant.](https://app.codacy.com/gh/WebHeroSchool/js_test/pullRequest?prid=5502397)